src/native/eventpipe/ds-eventpipe-protocol.h

```diff
@@ -36,6 +38,8 @@ struct _EventPipeCollectTracingCommandPayload_Internal {
 	dn_vector_t *provider_configs;
 	uint32_t circular_buffer_size_in_mb;
 	EventPipeSerializationFormat serialization_format;
+	bool rundown_requested;
+	bool disable_stacktrace;
```

Comment:
We discussed the naming earlier (https://github.com/dotnet/runtime/pull/84077#discussion_r1177478383): the intent is to make the interaction between the env variable and the IPC parameter clearer. By default the stacks are enabled, the env variable disables it completely (as if the stacktrace feature didn't exist at all), and the IPC parameter can disable it per-session. If we name it `enable_stackwalk` it will create confusion because it won't enable the stacktrace when the env variable is set.

I see your point about the double negatives, for me it's fine to have `request_stackwalk` parameter which doesn't behave confusingly (i.e. you can ask for the stackwalk but it doesn't mean it will always be granted). Please see with @lateralusX if he's ok with that :)

Reply:
I re-read and I see that I misspoke there, sorry for the confusion. I thought David was disagreeing with @lateralusX and then I when said I felt the same that meant I too was disagreeing. Reading more carefully I see David said he was OK either way.

I do think the name should be postive rather than negative and I care because the name will be included in public APIs and documents. We generally do not take the presence of environment variable overrides into account when naming our parameters/settings. The expectation is that setting one of those env vars is (or will become) a rare scenario and we should optimize our naming for clarity in the common case when the env var is not set. @lateralusX are you OK with that?

Reply:
> @lateralusX are you OK with that?

Of course.